pkgs/development/python-modules/backgroundremover/default.nix

```diff
@@ -0,0 +1,78 @@
+{ buildPythonPackage
+, certifi
+, charset-normalizer
+, fetchPypi
+, ffmpeg-python
+, filelock
+, filetype
+, gdown
+, hsh
+, idna
+, lib
+, more-itertools
+, moviepy
+, numpy
+, pillow
+, pymatting
+, pysocks
+, requests
+, scikitimage
+, scipy
+, six
+, torch
+, torchvision
+, tqdm
+, urllib3
+, waitress
+, flask
+}:
+
+buildPythonPackage rec {
```

Comment:
It's also a library

Reply:
About this double behaviour there is the following:

> In case a package is used as both a library and an application, then the package should be in pkgs/top-level/python-packages.nix since only those packages are made available for all interpreter versions.
> https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#building-packages-and-applications-building-packages-and-applications

Reply:
BTW machine learning stuff break a lot in my experience when it's "homemade", like custom architectures (ex: dlib). Stuff that basically just wraps PyTorch rarely breaks unless there are some custom native extension modules.

Reply:
And that's why I love PyTorch, and now also transformers.

Stuff works without low level headache, it's wonderful, at least on CPU and NVidia.

Reply:
> About this double behaviour there is the following:

Thanks for letting me know. I know that because I have reviewed hundreds of python packages.
I'm talking about the document of backgroundremover.

> BTW machine learning stuff break a lot in my experience when it's "homemade", like custom architectures

It's not just extensions. I've seen and fixed a lot of libraries over the past year that don't keep up with updates of numpy, pillow, torch, etc.

> Stuff works without low level headache, it's wonderful, at least on CPU and NVidia.

This is because cuda maintainers and other maintainers are constantly paying attention.

Reply:
Done